libpod/runtime.go

```diff
@@ -15,7 +18,9 @@ type RuntimeOption func(*Runtime) error
 
 // Runtime is the core libpod runtime
 type Runtime struct {
-	// TODO populate
+	store         storage.Store
+	systemContext types.SystemContext
+	output        io.Writer
```

Comment:
I'm a little iffy about keeping this around for everything in the runtime... How many commands actually use it? Can we pass it into them individually, instead of keeping one for the entire runtime?

I can see us eventually replacing this with a per-command channel that returns a status ("40% downloaded", "download error", "complete") or similar for the final API